jekyll/_cci2/server-3-faq.adoc

```diff
@@ -6,21 +6,19 @@ version:
 = CircleCI Server v3.x FAQ
 :page-layout: classic-docs
 :page-liquid:
-:page-description: Find answers about the CircleCI server v3.x data retention policy, what control is granted over Nomad certificates, and application status options in KOTS.
```

Comment:
This file will remain for server v3 so I will revert changes. Let me know if that's not right though!